### ISSUE TITLE
Dataset - Distribution definitions. Properties "issued" and dataset_id

### ISSUE THREAD

**Issue by JacquemotMC** [RDA-DMP-Common/RDA-DMP-Common-Standard#56]
My thoughts about  Dataset versus Distribution in DMP Common Standard:

Dataset: provides the description of a logical (or scientifically consistent) dataset (empassing dataset as well as other research products). 
The dataset as described in a DMP may lead to several distributions.  

Distributions are the physical data that are actually available and distributed via repositories, which deliver PIDs. 
--> These definitions would be compliant with Dcat
--> But what if a dataset as defined in a DMP (based on scientific or managing purposes) leads to several distributions of distinct data that could be possibly shared in different repositories? 

In Dcat there is a property  "issued" (meaning available I presume)  for Distribution that could be used .
"issued" for Dataset : is it the date of the publication of metadata/description?

Dcat does not specify Distribution_id although access_url would be associated with  a DOI for example but it is still not a PID.

**Comment by jmartinkova** [comment]
I agree that the meaning of _issued_ for a _dataset_ is unclear. It would be helpful to clarify its definition—does it refer to the dataset's creation date or its last modification date? This is particularly important because the term _issued_ typically implies something has been released, published, or distributed. Releasing a _dataset_ usually means making it available through a _distribution_. However, I understand the need for compliance with DCAT, where this ambiguity also exists.

That said, I also support adding issued for distribution, as it is already included in DCAT v3 (and even v2), where _[dcat:Distribution](https://www.w3.org/ns/dcat#Distribution)_ has _[dcterms:issued](http://purl.org/dc/terms/issued)_ as a property. Since a _dataset_ typically has multiple _distributions_, and a _distribution_ represents a published _dataset_, it makes sense for _distributions_ to have a publication date.

My suggestions:
1. Add _issued_ for _distribution_
2. Clarify the meaning of _issued_ for _dataset_ in the description or consider removing it.

**Comment by MarekSuchanek** [comment]
As we want to be aligned with DCAT, the proposed solution is:

- Add `issued` for Distribution (according to DCAT)
- Clarify the meaning of issued for dataset in the description (done as part of #110)

Would that work?

@briri is it OK not to add `distribution_id`? We discussed that there is not typically such an ID, you have dataset ID, alt. related/alternate dataset IDs but distribution just carries the access/download URL.

**Comment by briri** [comment]
yes, we're not using a `distribution_id` currently and the introduction of the other related/alternate ids will cover the need

**Comment by MarekSuchanek** [comment]
@briri can you please check #140 and comment if you spot any issue there?
